modules/channel-web/src/messaging-migration/down-sqlite.ts

```diff
@@ -21,35 +21,47 @@ export class MessagingSqliteDownMigrator extends MessagingDownMigrator {
   protected async migrate() {
     await super.migrate()
 
+    const batchSize = this.bp.database.isLite ? 100 : 5000
     const convCount = <number>Object.values((await this.trx('msg_conversations').count('*'))[0])[0] || 0
     this.convoIndex = <number>Object.values((await this.trx('web_conversations').max('id'))[0])[0] || 1
 
-    for (let i = 0; i < convCount; i += 100) {
+    for (let i = 0; i < convCount; i += batchSize) {
       const convos = await this.trx('msg_conversations')
         .select('*')
         .offset(i)
-        .limit(100)
+        .limit(batchSize)
 
-      // We migrate 100 conversations at a time
+      // We migrate batchSize conversations at a time
       await this.migrateConvos(convos)
     }
   }
 
   protected async cleanup() {
     await super.cleanup()
 
-    await this.trx.raw('PRAGMA foreign_keys = OFF;')
-
-    await this.trx.schema.dropTable('msg_messages')
-    await this.trx.schema.dropTable('msg_conversations')
-    await this.trx.schema.dropTable('msg_users')
-    await this.trx.schema.dropTable('msg_clients')
-    await this.trx.schema.dropTable('msg_providers')
-
-    await this.trx.raw('PRAGMA foreign_keys = ON;')
+    if (this.bp.database.isLite) {
```

Comment:
That's kind of hackish and weird but okay!

Reply:
It is but it works